File: Client/src/app/classes/Trip.ts
import { Time } from "@angular/common";

export class Trip{
    constructor(public TripCode:number, public TripDestination?:String, public TypeCode?:Number, public Date?:Date, public DepartureTime?:Time, public TripDurationHours?:Number, public NumberOfAvailablePlaces?:Number, public Price?:Number, public NameType?:String, public Need_a_medic?:Boolean){}
}

File: Server/DAL/Travels/Trip.cs
using System;
using System.Collections.Generic;

namespace DAL.Travels;

public partial class Trip
{
    public int TripCode { get; set; }

    public string? TripDestination { get; set; }

    public int? TypeCode { get; set; }

    public DateTime? Date { get; set; }

    public TimeSpan? DepartureTime { get; set; }

    public int? TripDurationHours { get; set; }

    public int? NumberOfAvailablePlaces { get; set; }

    public decimal? Price { get; set; }

    public string? Photo { get; set; }

    public virtual ICollection<BookingPlace> BookingPlaces { get; set; } = new List<BookingPlace>();

    public virtual TravelType? TypeCodeNavigation { get; set; }
}
